dev-go/go-licenses/go-licenses-9999.ebuild

```diff
@@ -0,0 +1 @@
+go-licenses-0_pre20200426.ebuild
```

Comment:
We do not allow symlinks in ::gentoo repo. Are you sure you want to maintain -9999 versions of these ebuilds? 

Don't version the ebuild by the date when you wrote it, but by the date the commit was made you point to. As I see, it should be 20200227.

Reply:
symlinks - done

date of commit - done